src/yutto/api/cheese.py

```diff
@@ -129,7 +129,10 @@ async def get_cheese_subtitles(
     subtitles_info = subtitles_json_info["data"]["subtitle"]
     results: list[MultiLangSubtitle] = []
     for sub_info in subtitles_info["subtitles"]:
-        subtitle_text = await Fetcher.fetch_json(ctx, client, "https:" + sub_info["subtitle_url"])
+        subtitle_url = sub_info["subtitle_url"]
+        if not subtitle_url or subtitle_url.strip() == "":
```

Comment:
```suggestion
        if not subtitle_url.strip():
```

是否可以这样？还是说 `not subtitle_url` 是为了避免 `None` 的出现？如果是避免 `None` 的出现麻烦改为

```python
if subtitle_url is None or not subtitle_url.strip():
```

Reply:
另外这里可以添加一个 warning，说明某种语言的字幕链接有问题，跳过了